.github/actions/multistrap/action.yml

```diff
@@ -20,13 +20,11 @@ runs:
       run: |
         set -xe
 
-        multistrap_conf=""
-        if [ "${{ inputs.arch }}" = "armv7" ]; then
-          multistrap_conf=multistrap_raspbian_buster.conf
+        multistrap_conf="${{ inputs.multistrap-config }}"
+        if [ "$multistrap_conf" = "multistrap_raspbian_buster.conf" ]; then
```

Comment:
why? it's the same keyring, no need to complexify

Reply:
The question is do we really need this keyring for aarch64? We didn’t use it for 3.7 (aarch64)… that’s why there is a if statement here. If we indeed need it for 3.9 aarch64 then we don’t need the if statement to set the keyring.

Reply:
If you look at my config file for 3.9 aarch64, I don’t use it and it still works. So we don’t need it for aarch64 (any python version).

https://github.com/wasertech/STT/blob/8284e23c7acd3345f7cd9ea78be14ead8ec39a4f/.github/actions/multistrap/action.yml#L29